### ISSUE TITLE
Faster (keyboard) navigation in views

### ISSUE THREAD

**Issue by richardolsson** [zetkin/organize.zetk.in#1184]
Add keyboard navigation to views, similarly to how you navigate a spreadsheet. Some ideas on what should be possible:

1. Navigate up/down/left/right, e.g. with arrow keys
2. Enter edit mode to edit cells (#1111)
3. Toggle tag cells
4. Open person pane from row
5. Open linked information in cell (survey, note feed etc)
